lib/resources/Account.js

```diff
@@ -2,12 +2,40 @@
 
 var StripeResource = require('../StripeResource');
 var stripeMethod = StripeResource.method;
+var utils = require('../utils');
 
 module.exports = StripeResource.extend({
-  path: 'account',
+  // Since path can either be `account` or `accounts`, support both through stripeMethod path
```

Comment:
The semantics of using v1/accounts v. v1/account are pretty different, right? Like it'd be weird to see all these methods available but still be using the /account singleton retrieve. My preference is actually to create a new resource called "Accounts" for the new stuff, so you need to explicitly switch over.

cc @jimdanz since this was brought up during review but I'm not sure we reached a conclusion.